dotnet new streamdeck-plugin fails restore step

**Describe the bug**

The command `dotnet new streamdeck-plugin -n FirstPlugin` fails with a restore error.

**To Reproduce**

Steps to reproduce the behavior:
1. Run the command `dotnet new streamdeck-plugin -n FirstPlugin`

**Expected behavior**

The project should be created and the restore should succeed.

**Error Details**

```bash
dotnet new streamdeck-plugin -n FirstPlugin
The template "Stream Deck Plugin" was created successfully.

Processing post-creation actions...
Running 'dotnet restore' on FirstPlugin/FirstPlugin.csproj...
  Restoring packages for /Users/bobby/src/notmyself/FirstPlugin/FirstPlugin.csproj...
/Users/bobby/src/notmyself/FirstPlugin/FirstPlugin.csproj : error NU1102: Unable to find package StreamDeckLib with version (>= 1.0.0)
/Users/bobby/src/notmyself/FirstPlugin/FirstPlugin.csproj : error NU1102:   - Found 14 version(s) in nuget.org [ Nearest version: 0.2.354 ]
  Generating MSBuild file /Users/bobby/src/notmyself/FirstPlugin/obj/FirstPlugin.csproj.nuget.g.props.
  Generating MSBuild file /Users/bobby/src/notmyself/FirstPlugin/obj/FirstPlugin.csproj.nuget.g.targets.
  Restore failed in 1.2 sec for /Users/bobby/src/notmyself/FirstPlugin/FirstPlugin.csproj.

Restore failed.
Post action failed.
Description: Restore NuGet packages required by this project.
Manual instructions: Run 'dotnet restore'
```

**Screenshots**
If applicable, add screenshots to help explain your problem.

**Desktop (please complete the following information):**
 - OS: ALL
 - Version: Dev branch

--- a/Templates/StreamDeck.PluginTemplate.Csharp/content/_StreamDeckPlugin_.csproj
+++ b/Templates/StreamDeck.PluginTemplate.Csharp/content/_StreamDeckPlugin_.csproj
@@ -10,7 +10,7 @@
    <ItemGroup />
 
    <ItemGroup>
-      <PackageReference Include="StreamDeckLib" Version="1.*" />
+      <PackageReference Include="StreamDeckLib" Version="0.*" />
       <PackageReference Include="McMaster.Extensions.CommandLineUtils" Version="2.3.0" />
       <PackageReference Include="Microsoft.Extensions.Logging" Version="2.2.0" />
       <PackageReference Include="Newtonsoft.Json" Version="12.0.1" />

